packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -69,7 +68,7 @@ export const useMultisessionActions = (opts: UseMultisessionActionsParams) => {
 
   const handleSessionClicked = (session: ActiveSessionResource) => async () => {
     card.setLoading();
-    return setActive({ session, redirectUrl: opts.afterSignInUrl }).finally(() => {
+    return setActive({ session, redirectUrl: opts.afterSwitchSession }).finally(() => {
```

Comment:
If I understand things correctly, `useMultisessionActions` is called with `...useUserButtonContext()` in `UserButtonPopover`. In the user context, this seems to be called `afterSwitchSessionUrl` but here we are using `afterSwitchSession`, is this intentional?

(Not related to this PR: We seem to be calling `useUserButtonContext` twice in `UserButtonPopover`, it doesn't look very expensive but one could probably be cleaned up.)

Reply:
thanks for identifying this one.